### ISSUE TITLE
ERROR EOF

### ISSUE THREAD

**Issue by apopa116** [keybase/client#3632]
→ keybase login
▶ INFO Forking background server with pid=24860
Your keybase username or email address: apopa
▶ ERROR EOF

my log id: 79d0ce30d961c3fcffcf6d1c

**Comment by maxtaco** [comment]
Did you `run_keybase`?

**Comment by apopa116** [comment]
Yes I did. I tried killing all the processes, uninstalling keybase, installing it again. Same thing.

**Comment by maxtaco** [comment]
cc: @oconnor663, no service running on linux...

**Comment by oconnor663** [comment]
@apopa116 I just landed a PR that should hopefully fix you. Would you please give it a try after the next update goes out (hopefully Monday)?
